assets/pi.yaml

```diff
@@ -0,0 +1,23 @@
+---
+
+- type: application
+  name: PI
+  organization: Inflection AI
+  description: Personal AI chatbot designed to be conversational and specialized
+    in emotional intelligence.
+  created_date: 2023-05-02
+  url: https://inflection.ai/press
+  dependencies: []
```

Comment:
See the more recent blog posts, I think they say it is built using the `Inflection-1` model.

And then add Inflection-1 and document it.